File: pl-domain/profession-service/src/main/java/com/polylab/profession/ProfessionServiceApplication.java
package com.polylab.profession;

import com.jxareas.jxcore.annotations.EnableJxCore;
import com.jxareas.jxcore.annotations.EnableNonRestrictiveCorsPolicy;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@EnableJxCore
@EnableNonRestrictiveCorsPolicy
@SpringBootApplication
public class ProfessionServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(ProfessionServiceApplication.class, args);
    }

}

File: pl-domain/profession-service/src/main/java/com/polylab/profession/ProfessionServiceApplication.kt
package com.polylab.profession;

import com.jxareas.jxelerator.annotations.EnableJxelerator
import com.jxareas.jxelerator.annotations.EnableNonRestrictiveCorsPolicy
import com.jxareas.jxelerator.extensions.runApp
import org.springframework.boot.autoconfigure.SpringBootApplication

@EnableJxelerator
@EnableNonRestrictiveCorsPolicy
@SpringBootApplication
open class ProfessionServiceApplication

fun main(vararg args : String) = runApp(ProfessionServiceApplication::class, *args)
